frontend/src/components/App.js

```diff
@@ -4,51 +4,115 @@ import './App.css';
 import Grid from "@material-ui/core/Grid";
 import DatePicker from "./DatePicker";
 import CalendarLink from "./CalendarLink";
-
-//todo get from API
-const CALENDAR_LINK = 'https://calendar.google.com/calendar?cid=MXQ0NDRob24wdTE1cGkxOWlyYzUxaTgxb3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ';
-const CALENDAR_ID = '1t444hon0u15pi19irc51i81os@group.calendar.google.com';
+import CalendarSelect from "./CalendarSelect";
 
 //todo move to prop file
-const ADDRESS = 'http://192.168.88.254:5000';
-const API = '/api/calendar';
+const ADDRESS = 'http://192.168.88.254:5000/';
```

Comment:
Its better to use a proxy middleware
https://create-react-app.dev/docs/proxying-api-requests-in-development/